== SUBMITION (2019-03-30 19:16:21) 
-- CODE:
print(180\4)
-- EXECUTION TIME:
0.149348
-- ERROR:
  File "/home/codebench/codes/1577/1329/main.py", line 1
    print(180\4)
               ^
SyntaxError: unexpected character after line continuation character

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-30 19:20:18) 
-- CODE:
print(180/4)
-- EXECUTION TIME:
0.198112
-- TEST CASE 1:
---- input:
xxx
---- correct output:
45.0
---- user output:
45.0
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
